Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -1,24 +1,78 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace BootCamp.Chapter
 {
     public class CredentialsManager
     {
         private readonly string _credentialsFile;
 
         public CredentialsManager(string credentialsFile)
         {
+            if (string.IsNullOrEmpty(credentialsFile))
+            {
+                throw new ArgumentException();
```

Comment:
nameof should be added to ArgumentExcpeiton

Reply:
What would the nameof do?